fawltydeps/packages.py

```diff
@@ -131,19 +136,45 @@ def packages(self) -> Dict[str, Package]:
         This enumerates the available packages  _once_, and caches the result for
         the remainder of this object's life in _packages.
         """
```

Comment:
I think the documentation of this function should be updated.

Reply:
Changed to:
```
        """Gather a custom mapping given by a user.
        
        Mapping may come from two sources:
        * _mapping_path_ which is a file in a given path, which is parsed to a ditionary
        * _custom_mapping_ which is a dictionary of package to imports mapping.
  ```****